scripts/diag/rpc-volume-report.py

```
@@ -0,0 +1,227 @@
+          lambda t: f"{human_bytes(t['bytes'])} "
+                    f"({100 * t['bytes'] / total_bytes:4.1f}%)")
+
+    table("by DECODE TIME -- what is actually costing CPU:",
+          lambda t: t["decodems"],
+          lambda t: f"{t['decodems'] / 1000:8.1f}s "
+                    f"({100 * t['decodems'] / decode_ms:4.1f}%)" if decode_ms else "n/a")
+
+    if rollup["n"]:
+        byte_pct = 100 * rollup["bytes"] / total_bytes if total_bytes else 0.0
+        time_pct = 100 * rollup["decodems"] / decode_ms if decode_ms else 0.0
+        print(f"\nspread thin, outside both per-window rankings "
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)